data/darktableconfig.xml.in

```diff
@@ -200,6 +200,38 @@
     <shortdescription>enable disk backend for full preview cache</shortdescription>
     <longdescription>if enabled, write full preview to disk (.cache/darktable/) when evicted from the memory cache.\nnote that this can take a lot of memory (several gigabytes for 20k images) and will never delete cached thumbnails again.\nit's safe though to delete these manually, if you want.\nlight table performance will be increased greatly when zooming image in full preview mode.</longdescription>
   </dtconfig>
+  <dtconfig prefs="processing" section="cpugpu" restart="true">
```

Comment:
The processing section is more for the dev/darkroom settings. I would put those into the lighttable section maybe?

As we are at it the two prefs in processing:
- enable disk backend for thumbnail cache
- enable disk backend for full preview cache

Could be moved into thumbnails section in lighttable tab. So those two prefs plus the new ones here would fit together under lighttable/thumbnails (which seems natural to me).

To be discussed... What do you think?

Reply:
I think that would be indeed better. Will do